src/app/example_configs/app-logging.yaml

```diff
@@ -16,7 +16,7 @@ handlers:
     encoding: utf8
     formatter: simple
     maxBytes: 10485760
-    filename: "beer-garden.log"
+    filename: "/var/log/beer-garden/beer-garden.log"
```

Comment:
Hard coding this to `/var/log` removes the ability to run BG from the command line using this file without root access. I'd prefer if the file could be used for both scenarios without editing. Perhaps, `tmp` instead?

Reply:
Yeah, this felt bad when I did it, and then I guess I forgot to go back and figure out what I really wanted to do here.  I reverted this back to what it was and instead generate a new app-logging.yaml during the docker build which is configured to output just to stdout, which seems like the right default config in the context of a docker container.